### DISCUSSION TITLE
How do you import the project into a CMake project in Visual Studio 2019?

### DISCUSSION THREAD

**Discussion by enjoysmath** [s-martin/galib#37] [Q&A] [answered]
The directions say, that's simple, just do this:

> Visual Studio 2019
> Open path in Visual Studio 2019 as CMake project.

However, it is not that simple.  I'm struggling to do this.  I have CMake support for Visual Studio installed.

I can create a CMake project.  I can open a CMake folder, but doing so does not create a project for you.  And the instructions on Microsoft don't explain what to do here.

**Comment by s-martin** [comment, marked as answer]
Do you use Visual Studio 2019? 
The description here should work:
https://docs.microsoft.com/en-us/cpp/build/cmake-projects-in-visual-studio?view=msvc-160#ide-integration

You need to select the root directory of this project, which you have cloned locally.

A CMake project in VS doesn't create a regular VS project, but uses the CMake tool chain. A menu item "CMake" should appear. See also the link above for details.
